python/src/aiconfig/eval/lib.py

```diff
@@ -90,7 +91,7 @@ def __str__(self) -> str:
 
 
 async def _evaluate_for_sample(
-    eval_params: SampleEvaluationParams[T_InputDatum, T_OutputDatum, T_MetricValue]
+    eval_params: SampleEvaluationParams[T_InputDatum, T_OutputDatum, T_MetricValue], timeout: int
```

Comment:
would recommend renaming the variable to include the time unit. For e.g. `timeout_ms`

Reply:
Added here https://docs.google.com/document/d/15gx4vVmKVgI5bDHjSDd8xCQmTGcC8RrtLU0iCa32AIQ/edit